src/appleseed/renderer/modeling/project/projectfilewriter.h

```diff
@@ -64,11 +57,18 @@ class APPLESEED_DLLSYMBOL ProjectFileWriter
     };
 
     // Write a project to disk. Return true on success, false otherwise.
-    // Write project as project file or as packed project based on options
+    // Write project as project file or as packed project based on options.
     static bool write(
         const Project&  project,
         const char*     filepath,
         const int       options = Defaults);
+
+  private:
+    // Write a project file to disk. Return true on success, false otherwise.
+    static bool write_project_file(
+      const Project&  project,
```

Comment:
So, the general rule for indentation is 4 spaces, no tabs (ever).

The only things that are indented with 2 spaces are:

- initializer and inheritance lists
- `public`, `protected`, `private`, `case` and `default` keywords

Everything else is indented with 4 spaces.

Example:

    class Derived
      : public Base
    {
      public:
        void foo(
            const int y,
            const int z);
      
      private:
        int bar(const int x)
        {
            switch (x)
            {
              case 1: return 2;
              case 2: return 4;
              default: return 0;
            }
        }
    };